<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>760</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QPushButton" name="RegisterButton_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>670</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
      background-color: rgba(255, 255, 255, 0.35);
      border: none; 
      border-radius: 10px; 
      color: #f3eabd;
      padding: 5px 10px; 
      transition: background-color 0.35s linear;
  } 
QPushButton:hover {
	   color:  #c68359;
       background-color: #f3eabd;
       border: none; 
 }</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="BgUnder_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>480</width>
       <height>400</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: #c68359;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="BgBelow_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>480</width>
       <height>360</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>400</width>
       <height>380</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
   border: none;             
   border-radius: 25px;           
   background-color: white;        
   margin-top: 5px;                
}


</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <widget class="QPushButton" name="LoginButton_1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
		color: white;
		border: none;
		border-radius: 25px;
		background-color: #c68359;
}
QPushButton:hover {
		color: #c68359;
		border: 2px solid #c68359;
		background-color: white;
}</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="Logo_1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>80</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>341</width>
        <height>154</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_1">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QGroupBox" name="usernameGB_1">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
	   border: 2px solid #f9d6c2;
  	   border-radius: 30px;
       background-color: transparent;
} </string>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="username_1">
            <property name="text">
             <string>username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="userLine_1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	border: none;
	font-size: 16px;
	background: transparent;
	color: black;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="passwordGB_1">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
	   border: 2px solid #f9d6c2;
  	   border-radius: 30px;
       background-color: transparent;
} </string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="password_1">
            <property name="text">
             <string>password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passLine_1">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	border: none;
	font-size: 16px;
	background: transparent;
	color: black;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <zorder>BgUnder_1</zorder>
    <zorder>RegisterButton_1</zorder>
    <zorder>BgBelow_1</zorder>
    <zorder>groupBox_1</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="BgUnder_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>480</width>
       <height>400</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: #c68359;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>400</width>
       <height>500</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
   border: none;             
   border-radius: 25px;           
   background-color: white;        
   margin-top: 5px;                
}


</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <widget class="QPushButton" name="RegisterButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
		color: white;
		border: none;
		border-radius: 25px;
		background-color: #c68359;
}
QPushButton:hover {
		color: #c68359;
		border: 2px solid #c68359;
		background-color: white;
}</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="Logo_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>80</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>341</width>
        <height>300</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QGroupBox" name="usernameGB_2">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
	   border: 2px solid #f9d6c2;
  	   border-radius: 30px;
       background-color: transparent;
} </string>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="username_2">
            <property name="text">
             <string>username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="userLine_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	border: none;
	font-size: 16px;
	background: transparent;
	color: black;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="EmailGB">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
	   border: 2px solid #f9d6c2;
  	   border-radius: 30px;
       background-color: transparent;
} </string>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item alignment="Qt::AlignmentFlag::AlignLeft">
           <widget class="QLabel" name="email">
            <property name="text">
             <string>username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="emailLine">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	border: none;
	font-size: 16px;
	background: transparent;
	color: black;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="passwordGB_2">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
	   border: 2px solid #f9d6c2;
  	   border-radius: 30px;
       background-color: transparent;
} </string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="password_2">
            <property name="text">
             <string>password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passLine_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	border: none;
	font-size: 16px;
	background: transparent;
	color: black;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="confirmPasswordGB">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
	   border: 2px solid #f9d6c2;
  	   border-radius: 30px;
       background-color: transparent;
} </string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="confirmPass">
            <property name="text">
             <string>password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="confimPassLine">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	border: none;
	font-size: 16px;
	background: transparent;
	color: black;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="BgBelow_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>480</width>
       <height>360</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginButton_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>670</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
      background-color: rgba(255, 255, 255, 0.35);
      border: none; 
      border-radius: 10px; 
      color: #f3eabd;
      padding: 5px 10px; 
      transition: background-color 0.35s linear;
  } 
QPushButton:hover {
	   color:  #c68359;
       background-color: #f3eabd;
       border: none; 
 }</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <zorder>BgUnder_2</zorder>
    <zorder>BgBelow_2</zorder>
    <zorder>LoginButton_2</zorder>
    <zorder>groupBox_2</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
